src/core/actions/payloads.ts

```diff
@@ -67,14 +106,16 @@ namespace Payload {
     }
   }
 
+  export interface NavigationId {
```

Comment:
Would this be better put under the `Navigation` namespace and be called `Navigation.Id`?

Reply:
Makes sense